lib/install/validate/validator.go

```diff
@@ -902,3 +914,28 @@ func (v *Validator) syslog(op trace.Operation, conf *config.VirtualContainerHost
 		RAddr:   host,
 	}
 }
+
+// FIXME ATC DEBT setting this value needs to be moved to Dispatcher
+// https://github.com/vmware/vic/issues/6803
+// set PersistNetworkBacking key to "true"
+func (v *Validator) ConfigureVCenter(ctx context.Context) error {
+	op := trace.FromContext(ctx, "Set vCenter serial port backing")
+	defer trace.End(trace.Begin("", op))
+
+	errMsg := "Set vCenter settings SKIPPED"
+	if !v.sessionValid(op, errMsg) {
+		return nil
```

Comment:
Should we still return nil here?

Reply:
yes its just skipped and sessionValid handles the error message and noteError